cypress/e2e/signIn.cy.js

```diff
@@ -29,6 +28,10 @@ describe('Sign In page', () => {
   });
 
   it('should not provide an ability to log in with wrong credentials', () => {
+    signInPage.typeEmail('Gar' + user.email);
+    signInPage.typePassword(user.password);
+    signInPage.clickSignInBtn();
 
+    signInPage.assertErrorMessage();
```

Comment:
Ensure that the `assertErrorMessage` method in the `signInPage` object is correctly implemented to check for the specific error message displayed when login fails.